src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
```

Comment:
I guess you could shorten it with arrow funciton ▶️

Reply:
It's not possible to use arrow function here.

Reply:
@AndrzejPec That is not possible, because `this` keyword has different value within arrow functions.